import_flow_events.py

```diff
@@ -5,7 +5,7 @@
 import import_events
 
 TEMPORARY_SCHEMA = """
-    type VARCHAR(64) NOT NULL ENCODE zstd,
+    type VARCHAR(79) NOT NULL ENCODE zstd,
```

Comment:
Should this be length 80? `continue:flow_id` has 16 characters. I think it might be worthwhile to add a quick comment about it.